packages/runtime/container-runtime/src/summaryGenerator.ts

```diff
@@ -354,6 +354,7 @@ export class SummaryGenerator {
                 const retryAfterSeconds = summaryNack?.retryAfter;
 
                 const error = new LoggingError(`summaryNack: ${message}`, { retryAfterSeconds });
+                logger.sendTelemetryEvent({ eventName: "SummaryNack", ...generateTelemetryProps }, error);
```

Comment:
I do not think retryAfter[Seconds] is part of generateTelemetryProps, so it would be useful to add it to telemetry event.